packages/truffle-core/cli.js

```diff
@@ -38,16 +42,21 @@ if (userWantsGeneralHelp) {
 command.run(inputArguments, options, function(err) {
```

Comment:
I think you are probably already on it Faina, but you are going to have the version sent to GA as well, right?  Just wanted to add this comment here so we don't lose it.

Reply:
Yes, I send it! This is currently being added on inside of google-analytics.js before sending the event (it will be part of the error string if there is one, otherwise sent as an event label for init and unbox), though I'm refactoring that file to move the child process out into its own file.